Build.Dockerfile

```diff
@@ -27,6 +26,14 @@ RUN apt-get update \
     \
     && rm -r /var/lib/apt/lists/*
 
+# Install Ruby and RVM
+ RUN gpg --keyserver hkp://keys.gnupg.net:80 --keyserver-options http-proxy=${http_proxy} --recv-keys=${RECV_KEYS} && \
+     curl -sSL https://get.rvm.io | grep -v __rvm_print_headline | bash -s stable --ruby=ruby-2.7.2 && \
+     echo "source /usr/local/rvm/scripts/rvm" >> ~/.bashrc && \
+     # Install gems \
+     /bin/bash -l -c "gem install bundler liquid-cli:0.0.1 stf-client:0.3.0-rc.12 --no-document" && \
+     /bin/bash -l -c "gem update --system"
```

Comment:
Why not to make a separate image for this? Most of the builds that use Build image won't need this, and those that need this line don't use android sdk?